lib/tools/adb-commands.js

```diff
@@ -1724,4 +1724,20 @@ methods.performEditorAction = async function performEditorAction (action) {
   }
 };
 
+/**
+ * Get tz database time zone formatted timezone
+ *
+ * @returns {string} TZ database Time Zones format
+ *
+ * @throws {error} If any exception is reported by adb shell.
+ */
+methods.getCurrentTimeZone = async function getCurrentTimeZone () {
```

Comment:
why not just getTimeZone?

Reply:
Ah, I implemented with iOS one...
(It had default/system/current timezone)

Reply:
ah, no. I was confused with locale , haha